core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2571,12 +2557,17 @@ public String toString() {
               + "an exponential backoff. This property determines the maximum duration to retry for"
               + " before giving up.")
           .build();
+  /**
+   * @deprecated since version 1.8 and will be removed in version 2.0.
+   */
   public static final PropertyKey USER_RPC_RETRY_MAX_NUM_RETRY =
       new Builder(Name.USER_RPC_RETRY_MAX_NUM_RETRY)
+          .setAlias(new String[]{Name.MASTER_RETRY})
           .setDefaultValue(100)
           .setDescription("Alluxio client RPCs automatically retry for transient errors with "
               + "an exponential backoff. This property determines the maximum number of "
-              + "retries.")
+              + "retries. This property has been deprecated and please use time based retry with"
```

Comment:
needs a semicolon, dash, or period after "deprecated", and a space after "retry with"

Reply:
fixed